### PR TITLE
Quotes CSV loader

### PR DESCRIPTION
**Pull request by jodastephen**
Moved from strata-examples

### CODE REVIEW

File: modules/loader/src/main/java/com/opengamma/strata/loader/csv/QuotesCsvLoader.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+import com.opengamma.strata.collect.id.StandardId;
+import com.opengamma.strata.collect.io.CsvFile;
+import com.opengamma.strata.collect.io.ResourceLocator;
+import com.opengamma.strata.market.id.QuoteId;
+
+/**
+ * Loads a set of quotes into memory from CSV resources.
+ * <p>
+ * The quotes are expected to be in a CSV format, with the following header row:<br />
+ * {@code Date, Scheme, Ticker, Value}.
```
Comment (cjkent) [inline modules/loader/src/main/java/com/opengamma/strata/loader/csv/QuotesCsvLoader.java]: The headers in the comment are out of date. It's `Valuation Date` and `Symbology`

File: modules/loader/src/main/java/com/opengamma/strata/loader/csv/QuotesCsvLoader.java [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      ResourceLocator resource,
+      ImmutableMap.Builder<QuoteId, Double> builder) {
+
+    try {
+      CsvFile csv = CsvFile.of(resource.getCharSource(), true);
+      for (int i = 0; i < csv.rowCount(); i++) {
+        String dateText = csv.field(i, DATE_FIELD);
+        LocalDate date = LocalDate.parse(dateText);
+        if (date.equals(marketDataDate)) {
+          String schemeStr = csv.field(i, SYMBOLOGY_FIELD);
```
Comment (cjkent) [inline modules/loader/src/main/java/com/opengamma/strata/loader/csv/QuotesCsvLoader.java]: Variable name `schemeStr` doesn't make much sense now it's called symbology.

**Comment by cjkent** [comment]
That's everything from me

**Comment by jodastephen** [comment]
Fixes made, thanks

**Pull request merged**